webpack/stripes-config-plugin.js

```diff
@@ -8,52 +7,54 @@ const serialize = require('serialize-javascript');
 // This webpack plugin generates a virtual module containing the stripes configuration
```

Comment:
Maybe move these comments along to the exported class below, or to the top of the file.  Here its getting mashed up with another comment describing a function.

Reply:
Comments moved to top of file.